src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtFacilityServiceTest.java

```diff
@@ -0,0 +1,222 @@
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.entity.*;
+import uk.gov.hmcts.dts.fact.exception.DuplicatedListItemException;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.repositories.CourtRepository;
+import uk.gov.hmcts.dts.fact.repositories.FacilityTypeRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+@ExtendWith({SpringExtension.class, MockitoExtension.class})
+@ContextConfiguration(classes = AdminCourtFacilityService.class)
+public class AdminCourtFacilityServiceTest {
+
+    private static final int FACILITY_COUNT = 3;
+    private static final String COURT_SLUG = "some slug";
+    private static final String FACILITY_TYPE_NAME = "name";
+    private static final List<Facility> COURT_FACILITIES = new ArrayList<>();
+    private static final String NOT_FOUND = "Not found: ";
+    private static final String DUPLICATE_ERROR_MESSAGE = "Facility already exists: Facility3";
+    private static final String FACILITY_1 = "Facility1";
+    private static final String FACILITY_2 = "Facility2";
+    private static final String FACILITY_3 = "Facility3";
+    private static final String FACILITY_4 = "Facility4";
+    private static final String DESCRIPTION_1 = "Description1";
+    private static final String DESCRIPTION_CY_1 = "DescriptionCy1";
+    private static final String DESCRIPTION_2 = "Description2";
+    private static final String DESCRIPTION_CY_2 = "DescriptionCy2";
+    private static final String DESCRIPTION_3 = "Description3";
+    private static final String DESCRIPTION_CY_3 = "DescriptionCy3";
+    private static final String DESCRIPTION_4 = "Description4";
+    private static final String DESCRIPTION_CY_4 = "DescriptionCy4";
+    private static final uk.gov.hmcts.dts.fact.model.admin.Facility FACILITY = new uk.gov.hmcts.dts.fact.model.admin.Facility(FACILITY_4,DESCRIPTION_4,DESCRIPTION_CY_4);
+
+    @Autowired
+    private AdminCourtFacilityService adminCourtFacilityService;
+
+    @MockBean
+    private CourtRepository courtRepository;
+
+    @MockBean
+    private FacilityTypeRepository facilityTypeRepository;
+
+    @Mock
+    private Court court;
+
+    @Mock
+    private FacilityType facilityType;
+
+    @BeforeEach
+     void init() {
```

Comment:
There is indentation issue either on this line or the line before

Reply:
fixed indentation issue